oak/server/asylo/asylo_oak_app_main.cc

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2018 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <csignal>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "absl/flags/flag.h"
+#include "absl/flags/parse.h"
+#include "absl/synchronization/notification.h"
+#include "absl/time/time.h"
+#include "asylo/util/logging.h"
+#include "include/grpcpp/server.h"
+#include "include/grpcpp/server_builder.h"
+#include "oak/common/app_config.h"
+#include "oak/common/utils.h"
+#include "oak/proto/manager.grpc.pb.h"
+#include "oak/server/asylo/asylo_oak_manager.h"
+
+ABSL_FLAG(std::string, enclave_path, "", "Path of the enclave to load");
+ABSL_FLAG(std::string, module, "", "File containing the compiled WebAssembly module");
+ABSL_FLAG(std::string, storage_address, "127.0.0.1:7867",
```

Comment:
As an aside, should we use more IPv6 flags? I wonder whether IPv6 is even supported on GCP though, could you check, and then create an issue to use IPv6 everywhere in our code?

Reply:
If I understand correctly, we only can create external IPv6 addresses, that can be mapped to the internal IPv4 ones:
https://cloud.google.com/load-balancing/docs/ipv6